easybuild/easyconfigs/e/EvidentialGene/EvidentialGene-2022.01.14-gompi-2021b.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = "PackedBinary"
+
+name = "EvidentialGene"
+version = '2022.01.14'
+local_month = ['', 'jan', 'feb', 'mar', 'apr', 'may', 'jun', 'jul', 'aug', 'sep', 'oct', 'nov', 'dec']
+local_dlver = version.split('.')[0][-2:] + local_month[int(version.split('.')[1])] + version.split('.')[2]
+
+homepage = 'http://arthropods.eugenes.org/EvidentialGene/'
+description = """EvidentialGene is a genome informatics project for
+ "Evidence Directed Gene Construction for Eukaryotes",
+ for constructing high quality, accurate gene sets for
+ animals and plants (any eukaryotes), being developed by
+ Don Gilbert at Indiana University, gilbertd at indiana edu."""
+
+toolchain = {'name': 'gompi', 'version': '2021b'}
+
+source_urls = [
+    'http://arthropods.eugenes.org/EvidentialGene/other/evigene_old/',
+    'http://arthropods.eugenes.org/EvidentialGene/other/evigene_old/evigene_older/',
+]
+sources = ['evigene%s.tar' % local_dlver]
+checksums = ['d464240affb6d293f20dab440abe89381e580154502e777c020c0060c8947ba3']
+
+dependencies = [
+    ('Perl', '5.34.0'),
+    ('Exonerate', '2.4.0'),
+    ('CD-HIT', '4.8.1'),
+    ('BLAST+', '2.12.0'),
+]
+
```

Comment:
The Perl scripts, in `scripts` and subdirectories, have `#!/usr/bin/perl`. So we need a `fix_perl_shebang_for = ...` to be run on them.

Reply:
good catch, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/15576/commits/73a0f7c6d717d9ff9abd2e00e46784e941eb5d69